.buildkite/terraform-validate.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -ex
+
+cd "$(dirname "${BASH_SOURCE[0]}")"/..
+
+MODULES=(
+  modules/docker-mirror
+  modules/executors
+  modules/networking
+)
```

Comment:
I think we also want to validate `.`

Reply:
I don't think we can with modules

Reply:
But there's also terraform stuff in the root, is that not possible to lint at least?

Reply:
I spoke too soon (then deleted it)